### PR TITLE
UI / UX improvements

### PR DESCRIPTION
**Pull request by EdmundMai**
Taken from this doc:
https://docs.google.com/document/d/12NXPDbqZrxxAAC4dayx7Y5Kj4YFKmX2U1W7ymPWJnk8/edit

### CODE REVIEW

File: src/config/index.tsx [hunk 1]
```diff
@@ -3,8 +3,8 @@ export const IS_DEV = process.env.NODE_ENV === "development";
 const PROTOCOL = IS_DEV ? "http" : "https";
 
 const POLLING_NODE = IS_DEV
- // ? ["18.191.77.193"] // Travis broker
-  ? ["18.188.230.212"]
+  ? // ? ["18.191.77.193"] // Travis broker
```
Comment (CKH4) [inline src/config/index.tsx]: What is happening with these?

File: src/services/error-tracker.tsx [hunk 2]
```diff
@@ -1,12 +1,20 @@
 import Raven from "raven-js";
+import _ from "lodash";
```
Comment (CKH4) [inline src/services/error-tracker.tsx]: We probably shouldn't use lodash for this, but even if we do we shouldn't import everything.

Comment (EdmundMai) [reply src/services/error-tracker.tsx]: Don't think it's a big deal tbh

File: src/config/index.tsx (hunk 1 above)
Comment (EdmundMai) [reply src/config/index.tsx]: Prettier

**Pull request merged**